locales/en.json

```
@@ -1,7 +1,7 @@
 {
     "Title": "Contoso Air",
 	"Home": {
-		"HeroMessage": "Where do you<br>want to go?"
+		"HeroMessage": "Where do you really<br>want to go today?"
```

```suggestion
		"HeroMessage": "Where do you really<p>want to go today?"
```

I suggest we change it to a `<p>`instead of `<br>`? Change ready to be applied.

:octocat::heart: